src/tools/asm2wasm.cpp

```diff
@@ -115,6 +115,9 @@ int main(int argc, const char *argv[]) {
       .add("--emit-text", "-S", "Emit text instead of binary for the output file",
            Options::Arguments::Zero,
            [&](Options *o, const std::string &argument) { emitBinary = false; })
+      .add("--enable-atomics", "-a", "Enable the Atomics wasm feature",
```

Comment:
I called the flag `--enable-threads` in wabt, probably worthwhile to be consistent.

Reply:
Yeah, sounds good.